lib/views/multiline.coffee

```diff
@@ -0,0 +1,21 @@
+class MessageLine extends HTMLElement
```

Comment:
IMHO, We shouldn't just create extra element classes, especially when all they do is set current text. Also there is nothing wrong with setting classes, We both have been doing it for years and haven't found a problem with it, except that you have to replace `line-element` with `.line-element`.
 and IMO The simplicity is worth the extra lines. Also that's how we are doing in the base message views and it doesn't look ugly, does it?